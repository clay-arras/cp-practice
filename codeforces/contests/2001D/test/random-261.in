1
20
17 6 24 7 20 20 17 15 12 18 10 25 16 15 15 22 7 18 19 12
