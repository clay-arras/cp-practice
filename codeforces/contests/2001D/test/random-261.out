13
17 6 24 7 20 10 25 16 15 22 18 19 12
